rpcs3/Emu/Io/usb_device.cpp

```diff
@@ -228,6 +228,7 @@ void usb_device_emulated::control_transfer(u8 bmRequestType, u8 bRequest, u16 wV
 		default: sys_usbd.fatal("Unhandled control transfer(0): 0x%x", bRequest); break;
 		}
 		break;
+	case 0x80: sys_usbd.todo("Unimplemented control transfer: 0x%x", bmRequestType); break;
```

Comment:
It's not related to USIO.
Instead, the game was attempting to query the USB Descriptor of connected USB flash drives (VFS: dev_usbXXX), while there hasn't been such implementation in RPCS3 yet.

Reply:
Can you link me a log with `Log: {sys_usbd: Trace}` set?

Reply:
> Can you link me a log with `Log: {sys_usbd: Trace}` set?

Sorry, could you elaborate more on that?

Reply:
In config.yml there is a line `Log: {}`, change it to `Log: {sys_usbd: Trace}`.
Note that if you have a custom config for the game you need to change that in the custom config.

This will log all events Trace and above(ie everything) for sys_usbd.
I'm just curious how the game gets access to a device it didn't specifically query for?

Reply:
> In config.yml there is a line `Log: {}`, change it to `Log: {sys_usbd: Trace}`. Note that if you have a custom config for the game you need to change that in the custom config.
> 
> This will log all events Trace and above(ie everything) for sys_usbd. I'm just curious how the game gets access to a device it didn't specifically query for?

Here's the log:
[RPCS3.log](https://github.com/RPCS3/rpcs3/files/9791953/RPCS3.log)

Reply:
It is specific to USIO in this case (I don't know why you linked that to dev_usb, it's just querying the descriptor of the usio device afaict).

But I see we need a generic implementation of GET_DESCRIPTOR for usb_device_emulated in any case so it's fine to put it there.

Reply:
> It is specific to USIO in this case (I don't know why you linked that to dev_usb, it's just querying the descriptor of the usio device afaict).
> 
> But I see we need a generic implementation of GET_DESCRIPTOR for usb_device_emulated in any case so it's fine to put it there.

@RipleyTom I've implemented get_descriptor() for usb_device_emulated in PR #13214, please review. Thanks!